packages/cloudflare-optimizer-scripts/test/index.test.js

```diff
@@ -70,11 +71,31 @@ describe('handleEvent', () => {
     expect(await event.respondWith.mock.calls[0][0]).toBe(incomingResponse);
   });
 
+  it('should rewrite Location header for Redirect in response to Non-Get Requests (Proxy Mode)', async () => {
```

Comment:
@samouri 

It's not duplication but since the worker has different logic handling Get and Non-Get scenarios. We need to account that both scenarios redirects are rewritten properly